src/OpalCompiler-Core/CompilationContext.class.st

```diff
@@ -466,7 +481,8 @@ CompilationContext >> getClass [
 CompilationContext >> initialize [
 	options := Set new.
 	astTransformPlugins := OrderedCollection withAll: self class defaultTransformationPlugins.
-	semanticScope := OCMethodSemanticScope targetingClass: nil class
+	semanticScope := OCMethodSemanticScope targetingClass: nil class.
+	astParseTransformPlugins := OrderedCollection new: 2
```

Comment:
introduce `defaultParseTransformationPlugins` to be consistent with the other plugins?

Reply:
ok